ModelVale/ModelVale/ViewControllers/Main/ModelViewController.m

```diff
@@ -29,9 +47,37 @@ - (void)viewDidLoad {
     self.testButton.layer.cornerRadius = 10;
     self.trainButton.layer.cornerRadius = 10;
     self.dataButton.layer.cornerRadius = 10;
+    
+    self.numClusters = 2;
+    
+    [self.healthBarView initializeAnimationsWithDuration:animationDuration maxHealth:90 health:80];
+    [self.healthBarView animateFillingHealthBar:self.healthBarView.healthPath layer:self.healthBarView.healthShapeLayer];
+    
+    [self setHealthBarPropsForXP];
+    self.clusters = [self initializeXPClusters:self.numClusters avgNumPerCluster:20 seed:self.seed];
+    [self animateXPClusters:self.clusters];
+}
+
+- (void) viewWillAppear:(BOOL)animated {
+    [self reanimateXPClusters];
```

Comment:
remember to always call super when you override these functions ie - [super viewWillAppear:animated]